pkg/apiserver/handlers/webhook/validation_crd_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2023 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webhook
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	admv1 "k8s.io/api/admission/v1"
+)
+
+const (
+	contentTypeJson = "application/json"
+	contentTypeHtml = "text/html"
+)
```

Comment:
maybe consider having these in a separate file (`webhook_test.go`) since they are shared by all the other test files

Reply:
Moved to `webhook_test.go`